sources/launcher/Stride.Launcher/ViewModels/ReleaseNotesViewModel.cs

```diff
@@ -68,26 +70,17 @@ internal ReleaseNotesViewModel([NotNull] LauncherViewModel launcher, [NotNull] s
 
         public async void FetchReleaseNotes()
         {
-            string releaseNotesMarkdown = null;
+            string releaseNotesMarkdown;
 
             try
             {
-                var request = WebRequest.Create($"{BaseUrl}{ReleaseNotesFileName}");
-                using (var response = await request.GetResponseAsync())
+                using (var response = await httpClient.GetAsync($"{BaseUrl}{ReleaseNotesFileName}"))
```

Comment:
Can you check if the response is supposed to be explicitly disposed of for HttpClient - if so, we should do it in other places as well, if not, remove the using here.

Reply:
Yes, good point. Should be consistent one way or the other.

Reply:
It should be disposed so made it consistent.